### ISSUE TITLE
Mark required fields/tiles

### ISSUE THREAD

**Issue by agitator** [plone/plone.app.mosaic#123]
would a red tile label be enough?

**Comment by datakurre** [comment]
Agreed. This is also a bug, because if you try to save a page with validation error, the layout editor just "reloads" without showing any error (the error is hidden in properties dialog).
